control-plane/pkg/reconciler/broker/broker.go

```diff
@@ -349,6 +349,10 @@ func (r *Reconciler) finalizeKind(ctx context.Context, broker *eventing.Broker)
 		return controller.NewRequeueAfter(5 * time.Second)
 	}
 
+	return r.finalizeBrokerResources(ctx, broker, err, logger)
```

Comment:
The name is very generic and it doesn't much to "FinalizeKind" what's special about the method?

Reply:
oh, I removed the WIP :-)  this is a start... it's a bit messy func, the `finalizeKind`, hence this attempt

Reply:
Yeah, I like it, I was challenging to find a better name 

somethink like `finalizeKafkaClusterResources(ctx, logger, broker)` ?